pkg/cmd/flags.go

```diff
@@ -59,7 +59,7 @@ func AddWaitFlag(cmd *cobra.Command, wait *kube.WaitStrategy) {
 	cmd.Flags().Var(
 		newWaitValue(kube.HookOnlyStrategy, wait),
 		"wait",
-		"if specified, will wait until all resources are in the expected state before marking the operation as successful. It will wait for as long as --timeout. Valid inputs are 'watcher' and 'legacy'",
+		"wait until resources are ready (up to --timeout). Values: 'watcher' (default), 'legacy', and 'hookOnly'",
```

Comment:
Is the default "hookOnly"?

Reply:
I think it's quite complicated to understand it quickly.


| CLI Option              | Internal Wait Strategy |
|--------------------------|------------------------|
| `--wait=watcher`         | `watcher`              | 
| `--wait=legacy`          | `legacy`               | 
| `--wait=hookOnly`        | *Not possible*         | 
| *No `--wait` flag set*   | `hookOnly`             | 
| `--wait` (no value)      | `watcher`              | 
| `--wait=false`           | `hookOnly`             | 
| `--wait=true`            | `watcher`              | 

🤯